packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -2212,6 +2213,402 @@ curl -v -X GET \
+      },
+      "logo": [
+        {
+          "name": "medium",
+          "href": "https://tc2static.oktacdn.com/fs/bcg/4/gfs1wwhrwJR4LpB5X0w6",
+          "type": "image/png"
+        }
+      ],
+      "users": {
+        "href": "https://testorgone.okta.com/api/v1/apps/0oa7vicdkRNrz59R80w6/users"
+      },
+      "deactivate": {
+        "href": "https://testorgone.okta.com/api/v1/apps/0oa7vicdkRNrz59R80w6/lifecycle/deactivate"
+      }
+    }
+  },
+  {
+    "id": "0oa7vicvor8YSr9Hc0w6",
+    "name": "workday",
+    "label": "hrportal1",
+    "status": "ACTIVE",
+    "lastUpdated": "2021-05-17T23:10:22.000Z",
+    "created": "2021-05-17T23:10:22.000Z",
+    "accessibility": {
+      "selfService": false,
+      "errorRedirectUrl": null,
+      "loginRedirectUrl": null
+    },
+    "visibility": {
+      "autoSubmitToolbar": false,
+      "hide": {
+        "iOS": false,
+        "web": false
+      },
+      "appLinks": {
+        "login": true
+      }
+    },
+    "features": [],
+    "signOnMode": "SAML_2_0",
+    "credentials": {
+      "userNameTemplate": {
+        "template": "${source.login}",
+        "type": "BUILT_IN"
+      },
+      "signing": {
+        "kid": "wRejFXWxFlK9nnLozx5qKWQa3fg-JRXw7dvdlTjs5Pg"
+      }
+    },
+    "settings": {
+      "app": {
+        "siteURL": "https://acme.workday.com"
+      },
+      "notifications": {
+        "vpn": {
+          "network": {
+            "connection": "DISABLED"
+          },
+          "message": null,
+          "helpUrl": null
+        }
+      },
+      "signOn": {
+        "defaultRelayState": null,
+        "ssoAcsUrlOverride": null,
+        "audienceOverride": null,
+        "recipientOverride": null,
+        "destinationOverride": null,
+        "attributeStatements": []
+      }
+    },
+    "_links": {
+      "help": {
+        "href": "https://testorgone-admin.okta.com/app/workday/0oa7vicvor8YSr9Hc0w6/setup/help/SAML_2_0/external-doc",
+        "type": "text/html"
+      },
+      "metadata": {
+        "href": "https://testorgone.okta.com/api/v1/apps/0oa7vicvor8YSr9Hc0w6/sso/saml/metadata",
+        "type": "application/xml"
+      },
+      "appLinks": [
+        {
+          "name": "login",
+          "href": "https://testorgone.okta.com/home/workday/0oa7vicvor8YSr9Hc0w6/30",
+          "type": "text/html"
+        }
+      ],
+      "groups": {
+        "href": "https://testorgone.okta.com/api/v1/apps/0oa7vicvor8YSr9Hc0w6/groups"
+      },
+      "logo": [
+        {
+          "name": "medium",
+          "href": "https://tc2static.oktacdn.com/fs/bcg/4/gfs1wwhrwJR4LpB5X0w6",
+          "type": "image/png"
+        }
+      ],
+      "users": {
+        "href": "https://testorgone.okta.com/api/v1/apps/0oa7vicvor8YSr9Hc0w6/users"
+      },
+      "deactivate": {
+        "href": "https://testorgone.okta.com/api/v1/apps/0oa7vicvor8YSr9Hc0w6/lifecycle/deactivate"
+      }
+    }
+  }
+]
+```
+
+##### Request example: List apps with `name` eq `cornerstone`
```

Comment:
is it common to have two examples for the same filter? I am not sure what we gain by having an example for `workday` and for `cornerstone`

Reply:
@luishernandezcruz-okta - Thanks for pointing this out, let me check with @alifatolahi-okta and get back.

Reply:
can you please add a link to the example similar to how there is a link for the other filters:
<img width="997" alt="Screen Shot 2021-05-17 at 5 59 43 PM" src="https://user-images.githubusercontent.com/51870451/118574668-c15c4280-b739-11eb-8a30-ab8301a99330.png">

Reply:
@luishernandezcruz-okta - I spoke to @alifatolahi-okta about having examples for `workday` and for `cornerstone`. The idea is to ensure absolute clarity around what this new filter is to be used for i.e. to clearly differentiate between `'appName` vs `label`. Please see the original comment by @alifatolahi-okta around this - https://github.com/okta/okta-developer-docs/pull/2127#discussion_r633470651.

I will go ahead and add the example link as you pointed out.

Reply:
@luishernandezcruz-okta - just to follow up, I will be removing the 2nd example to stay consistent with the other examples.

Reply:
@luishernandezcruz-okta - Removed 2nd example, please review.